pyproject.toml

```
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "flexmock-plus"
+version = "0.10.6"
+description = "flexmock is a testing library for Python that makes it easy to create mocks, stubs and fakes."
+readme = "README.rst"
+homepage = "https://flexmock.readthedocs.io"
+repository = "https://github.com/flexmock/flexmock"
+license = "BSD License"
```

This is the recommended way to define the license (https://python-poetry.org/docs/pyproject#license):
```suggestion
license = "BSD-2-Clause"
```